### DISCUSSION TITLE
GET /sw.js and workbox after updated to v4.2.2

### DISCUSSION THREAD

**Discussion by IlirEdis** [vercel/next.js#64949] [App Router] [answered]
I recently updated to `v4.2.2` and i see a lot of requests to `GET /sw.js` in the terminal and also my front end console is full of some `workbox` messages (see screenshot).

<img width="1474" alt="Screenshot 2024-04-24 at 00 36 36" src="https://github.com/vercel/next.js/assets/128377763/6a22763a-f2d0-4397-844f-9349d8a3b10f">
<img width="494" alt="Screenshot 2024-04-24 at 00 46 08" src="https://github.com/vercel/next.js/assets/128377763/968ec483-446b-4c9a-8b71-afcc59f1ad29">

Is this normal? How can i stop displaying all those hundreds of messages in the console?

**Comment by icyJoseph** [comment, marked as answer]
Hi,

Have you used a Service Worker in another application? Or are you using `next-pwa` or something that implements a Service Worker?

Those logs come from this package: https://github.com/GoogleChrome/workbox/blob/9e69c4269c35e2db9fbba4d13e4e6206c7b66d2a/packages/workbox-strategies/src/utils/messages.ts#L15

**Reply by IlirEdis** [reply]
Yes, i tried to implement PWA with `next-pwa` but i uninstalled it and deleted the code responsible for that. I also deleted cookies in my browser but i still get it. 
I noticed that i get it only in the browser that i've been working on (Arc) and not in Chrome or any other browser.

**Reply by icyJoseph** [reply]
yeah you have to uninstall the Service Worker as well, look up how to do that in Arc. Getting rid of a Service Worker can be annoying...

**Reply by IlirEdis** [reply]
Great! It works. Thanks!

**Reply by icyJoseph** [reply]
Awesome! Please don't forget to mark as answered :)

**Reply by IlirEdis** [reply]
Sure thing :)
